README.md

```diff
@@ -250,6 +255,17 @@ We can run the kaniko executor image locally in a Docker daemon to build and pus
   ./run_in_docker.sh <path to Dockerfile> <path to build context> <destination of final image>
   ```
 
+**NOTE**: When running in Docker, Kaniko expects the absolute path to a 
```

Comment:
This is correct, but instead of saying `kaniko expects ... a Dockerfile path inside the container` we could say `run_in_docker.sh expects a path to a Dockerfile relative to the absolute path of the build context`. Since this section is about run_in_docker.sh that makes more sense here.